test/destroy.test.js

```diff
@@ -2,48 +2,46 @@
 
 const fs = require('fs')
 const SonicBoom = require('../')
-const { file, runTests } = require('./helper')
+const { file, runTests, once } = require('./helper')
 
 runTests(buildTests)
 
 function buildTests (test, sync) {
   // Reset the umask for testing
   process.umask(0o000)
 
-  test('destroy', (t) => {
-    t.plan(5)
+  test('destroy', async (t) => {
+    t.plan(4)
```

Comment:
Using `readFileSync`, we don't check the error because, in case of errors, the method throws them.